### PR TITLE
taschenrechner: init at 1.3.0

### PR DESCRIPTION
**Pull request by NetaliDev**
## Description of changes
Init `taschenrechner` at 1.3.0.

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 262603` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>taschenrechner</li>
  </ul>
</details>

**Review by Janik-Haag — APPROVED** [review]
LGTM

File: pkgs/by-name/ta/taschenrechner/package.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    rev = version;
+    hash = "sha256-PF9VCdlgA4c4Qw8Ih3JT29/r2e7i162lVAbW1QSOlWo=";
+  };
+
+  cargoHash = "sha256-SFgStvpcqEwus1JBs5ZyMHO1UD0oWV7mvS6o4v5gIFc=";
+
+  meta = with lib; {
+    description = "A cli-calculator written in Rust";
+    homepage = "https://gitlab.fem-net.de/mabl/taschenrechner";
+    license = licenses.gpl3;
```
Comment (OPNA2608) [inline pkgs/by-name/ta/taschenrechner/package.nix:23]: `gpl3` is [deprecated for being too unclear](https://github.com/NixOS/nixpkgs/commit/9d5bd8b835ddd62db5857b86fcda5af00c237bfa), can you clarify if `gpl3Only` or `gpl3Plus`?

**Review by OPNA2608 — CHANGES_REQUESTED** [review]

File: pkgs/by-name/ta/taschenrechner/package.nix (hunk 1 above)
Comment (Janik-Haag) [reply pkgs/by-name/ta/taschenrechner/package.nix:23]: oh yeah you are right.

Comment (Janik-Haag) [reply pkgs/by-name/ta/taschenrechner/package.nix:23]: The cargo.toml says gpl3.0 https://gitlab.fem-net.de/mabl/taschenrechner/-/blob/main/Cargo.toml#L5 so I guess gpl3Only

Comment (OPNA2608) [reply pkgs/by-name/ta/taschenrechner/package.nix:23]: Upstream sadly has neither a copyright note in their code files nor a copy of the license to clarify their exact wishes… `gpl3Only` would be my guess as well, just to be safe.

Comment (NetaliDev) [reply pkgs/by-name/ta/taschenrechner/package.nix]: Fixed

**Review by OPNA2608 — APPROVED** [review]
Result of `nixpkgs-review pr 262603` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>taschenrechner</li>
  </ul>
</details>

Runs fine, LGTM.

**Pull request merged**